test/run_test.py

```diff
@@ -893,6 +893,11 @@ def exclude_tests(exclude_list, selected_tests, exclude_message=None):
 
 
 def get_selected_tests(options):
+    if options.use_specified_test_cases == 'include':
+        options.include = list(SPECIFIED_TEST_CASES_DICT.keys())
+    elif options.use_specified_test_cases == 'bring-to-front':
+        options.bring_to_front = list(SPECIFIED_TEST_CASES_DICT.keys())
```

Comment:
So in get_test_case_args, I print a warning in the case where a test_module is passed in but it isn't in SPECIFIED_TEST_CASES_DICT. Could you either rephrase that comment since now, other test modules would still be allowed?

I'm also hesitant about this option currently, because say we have "test_nn": ["test1", "test1324"] in the dictionary. This option might mislead people into thinking that we will run the other tests in test_nn after we run test1 and test1324, but that's not true. The other test cases would not be run at all.